### PR TITLE
Consider text/plain when calling notebook.cell_output_text(idx)

### PR DESCRIPTION
**Pull request by nastra**
We have a use case where the cell output is some Pandas output and when
calling `notebook.cell_output_text(..)` we only get empty output, even
though `output["data"]["text/plain"]` contains actually the expected
text.

### CODE REVIEW

**Review by MSeal — CHANGES_REQUESTED** [review]
Thanks for the description of the problem and the improvement. Could we get an example input put into the tests to make sure we don't break this accidentally in the future? Thanks!

**Comment by nastra** [comment]
@MSeal sure thing. I added a small example with a test that shows the problem

**Comment by nastra** [comment]
@MSeal can you take another look at the proposed changes when you get a chance?

File: requirements-dev.txt [hunk 1]
```diff
@@ -3,6 +3,7 @@ coverage
 ipython
 ipykernel
 ipywidgets
+pandas==1.2.4
```
Comment (MSeal) [inline requirements-dev.txt:6]: Probably should be a `>=` dep but I can change that later

**Review by MSeal — APPROVED** [review]

**Comment by MSeal** [comment]
Thanks @nastra -- I'll look at shipping a patch release with the change so it's readily available.

**Pull request merged**